src/main/psxsdk/libsnd/ssstart.c

```diff
@@ -1,7 +1,98 @@
 #include "common.h"
 #include "libsnd_i.h"
 
-INCLUDE_ASM("main/nonmatchings/psxsdk/libsnd/ssstart", _SsStart);
+struct Thing {
+    u8 unk0;
+};
+extern struct Thing D_80032F00;
+
+void _SsSeqCalledTbyT_1per2();
+void _SsTrapIntrVSync();
+
+void _SsStart(s32 arg0) {
+    u16 rcnt_target;
+    s32 wait;
+    unsigned short var_v0_2;
+    u32 rcnt_spec;
+    u8 temp_a0;
+    void (*callback_func)();
+    s32 temp;
+    s32 temp2;
+    s32 temp3;
+    wait = 999;
+    do {
+        wait -= 1;
+    } while (wait >= 0);
+    D_80032F00.unk0 = 0;
+    D_80032F02 = 6;
```

Comment:
I wonder if `D_80032F02`,  `D_80032F01` and even `D_80032EFC` are part of the same struct as `D_80032F00`?